Juno-amr.Rmd

```diff
@@ -0,0 +1,123 @@
+```{}
+python3 juno-amr.py -s [species] -i [/mnt/scratch_dir/<my_folder>/fastq_or_fasta_file folder]
+```
+
+Please read the section [What to expect while running a Juno pipeline](#what-to-expect).  
+
+See the section [General Troubleshooting](#general-troubleshooting) for any problems you may encounter.  
+
+**Note:** Do not keep all your data (including results) on the scratch_dir partition. You can add the `--output` parameter to the command in order to place your output somewhere else.  
+
+### Output  
+
+In the output you can find 3 folders:  
+
+* **log:** Log with output and error file from the cluster for each snakemake rule/step that is performed.
+
+* **results_per_sample:** Output produced by ResFinder and PointFinder for each sample. CGE's explanation on the output can be found [here](https://cge.cbs.dtu.dk/services/ResFinder/output.php).
+
+* **summary:** Directory with 4 summary files created from each sample within the results_per_sample folder.
+    * **summary_amr_genes.csv:** Shows the samplename resistance gene, identity, alignment length/gene length and coverage per gene hit.
+
+    * **summary_amr_phenotype.csv:** Shows the type of match for each sample on the resistance for each antimicrobial.
+
+    * **summary_amr_pointfinder_predicition.csv:** - Shows the samplename and a 0(no hit) or 1(hit) prediciton for each mutation.
+
+    * **summary_amr_pointfinder_results** Shows the samplename, mutation, nucleotide change, amino acid change, resistance and PMID for each mutation hit
+
+
+**Note:** If you want your output to be stored in a folder with a different name or location, you can use the option `-o` or `--output` 
+
+```{}
+python3 juno-amr.py -i /mnt/scratch_dir/<my_folder>/<my_data>/ -species salmonella -o /mnt/scratch_dir/<my_folder>/<my_results>/
+```
+
+### Troubleshooting for this pipeline
+
+**Please read the [General Troubleshooting](#general-troubleshooting) section first!**
+
+#### Other problems or failing rules  
+
+The `r parameters$pipeline_name` pipeline is still in development which means that sometimes the process can fail.
+
+Before contacting for help, try these two steps:
+
+1. Re-run the pipeline again and see if the process continues. If it does, please keep re-running the pipeline until your analysis is finished or there is no longer progress. In this case, send an email after the pipeline is finished so I can troubleshoot the problem.
+
+2. [Download](https://github.com/RIVM-bioinformatics/Juno-amr) the pipeline again and start from the beginning of this [handbook](#downloading-instructions). Sometimes there is an issue that has been resolved in newer versions of the pipeline.
+
+If the pipeline still fails after these two steps, please inform me about the problem. Send an [e-mail](mailto:roxanne.wolthuis@rivm.nl) with the following content:
+
+* The log and error files that can be found in the output folder
+* The path to your input directory
+* The path to where the pipeline is installed
```

Comment:
I like this new format. We should change it in all the other 'chapters'

Reply:
Shall i put it in the other chapters?

Reply:
@wolthuisr yes, that would be nice! 😃